### PR TITLE
host f2bdb Patch 1

### PR DESCRIPTION
**Pull request by audioscavenger**
as discussed with @quietsy , created host f2bdb Patch 1 and Patch 2 to also read the host's f2bdb only if it's mounted in swag at the default location. simple and efficient, no documentation as requested

### CODE REVIEW

**Comment by quietsy** [comment]
Please combine to a single PR

File: root/dashboard/swag-f2b.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import json
 import sqlite3
+import sys
 
+f2bdb = "/config/fail2ban/fail2ban.sqlite3"
```
Comment (quietsy) [inline root/dashboard/swag-f2b.py:5]: Please use an environment variable and combine the results of swag's f2b with the custom f2b, there's no need to display the same section twice

Comment (quietsy) [reply root/dashboard/swag-f2b.py:5]: (And it won't require any changes to the php)

**Comment by audioscavenger** [comment]
voila all is good now. using docker env variables, includes the expand button.
image
compose variables and volume needed to activate it:

volumes:
  - /var/lib/fail2ban/fail2ban.sqlite3:/var/lib/fail2ban/fail2ban.sqlite3:ro
environment:
  - MOD_DASHBOARD_F2B_HOST_DB=/var/lib/fail2ban/fail2ban.sqlite3    # swag mounted path to the host's fail2ban database; default is /var/lib/fail2ban/fail2ban.sqlite3
  - MOD_DASHBOARD_F2B_MAX_LINES=30                                  # max IPs to show per jail; default is 30

File: root/dashboard/swag-f2b-host.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import json
```
Comment (quietsy) [inline root/dashboard/swag-f2b-host.py:1]: There's a lot of duplicate code both here and in the php, better change the current code to handle the host f2b

File: root/dashboard/swag-f2b-expand.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    MOD_DASHBOARD_F2B_MAX_LINES = os.environ['MOD_DASHBOARD_F2B_MAX_LINES']
+if "TZ" in os.environ:
+    TZ = os.environ['TZ']
+
+if not os.path.exists(MOD_DASHBOARD_F2B_HOST_DB):
+    exit()
+
+con = sqlite3.connect(MOD_DASHBOARD_F2B_HOST_DB)
+cur = con.cursor()
+results = cur.execute("""
```
Comment (quietsy) [inline root/dashboard/swag-f2b-expand.py:29]: The banned IPs can be returned together with the jails in the initial load

**Comment by quietsy** [comment]
Added the host f2b part under `https://github.com/linuxserver/docker-mods/pull/351`, I'll consider the UI changes separately, thanks for the effort!

**Pull request closed without merging**